algebras/algebra/src/main/scala/endpoints4s/algebra/ChunkedEntities.scala

```diff
@@ -27,39 +27,55 @@ package endpoints4s.algebra
   *
   * @group algebras
   */
-trait ChunkedEntities extends EndpointsWithCustomErrors {
+trait ChunkedEntities
+    extends EndpointsWithCustomErrors
+    with ChunkedRequestEntities
+    with ChunkedResponseEntities
+
+trait Chunks {
 
   /** A stream of chunks of type `A`.
     *
     * @tparam A Information carried by each chunk
     * @group types
     */
   type Chunks[A]
+}
 
+trait ChunkedRequestEntities extends Chunks {
```

Comment:
I remember we had a discussion about splitting algebra to request algebra and response algebra, but I couldn’t find it back. Could you please remember me why we need to split the algebra?

Reply:
It is because of browsers - there is no support in XHR and Fetch to deal with request body streams yet.